packages/editor/src/components/block-list/index.js

```diff
@@ -182,37 +193,54 @@ class BlockList extends Component {
 			blockClientIds,
 			rootClientId,
 			isDraggable,
+			selectedBlockClientId,
+			multiSelectedBlockClientIds,
 		} = this.props;
 
 		return (
 			<div className="editor-block-list__layout">
-				{ map( blockClientIds, ( clientId, blockIndex ) => (
-					<BlockListBlock
-						key={ 'block-' + clientId }
-						index={ blockIndex }
-						clientId={ clientId }
-						blockRef={ this.setBlockRef }
-						onSelectionStart={ this.onSelectionStart }
-						rootClientId={ rootClientId }
-						isFirst={ blockIndex === 0 }
-						isLast={ blockIndex === blockClientIds.length - 1 }
-						isDraggable={ isDraggable }
-					/>
-				) ) }
+				{ map( blockClientIds, ( clientId, blockIndex ) => {
+					return (
+						<AsyncModeProvider
+							key={ 'block-' + clientId }
```

Comment:
I know we had it previously, but I wonder why it matters to have the `key` be prefixed.

Reply:
It doesn't